packages/utils/src/display.ts

```diff
@@ -73,8 +73,10 @@ export function stringify(
     })
   }
 
+  // Prevents infinite loop https://github.com/vitest-dev/vitest/issues/7249
+  const nextMaxDepth = maxDepth === Infinity ? Number.MAX_SAFE_INTEGER : Math.floor(maxDepth / 2)
```

Comment:
MAX_NUMBER would still crash the browser, so Im using MAX_SAFE_INTEGER which works fine. If you want me to use any specific number, please send it here so I can replace